sync/class.jetpack-sync-actions.php

```diff
@@ -179,26 +241,32 @@ static function send_data( $data, $codec_name, $sent_timestamp, $queue_id, $chec
 		return $response;
 	}
 
-	static function do_initial_sync( $new_version = null, $old_version = null ) {
-		$initial_sync_config = array(
-			'options' => true,
-			'network_options' => true,
-			'functions' => true,
-			'constants' => true,
-		);
+	static function do_initial_sync( $new_version = null, $old_version = null, $network_site = false ) {
+		$initial_sync_config = self::get_update_full_sync_config();
 
 		if ( $old_version && ( version_compare( $old_version, '4.2', '<' ) ) ) {
```

Comment:
I think it is ok to leave it here. Otherwise we would need to pass in the old_version variable. Not sure if that makes sense anywhere but the upgrade part. But if you feel strongly about it I can change it